### PR TITLE
Show account name

### LINKED ISSUE #371: Add Account Name
In the android app is the name of the Account never shown, please add the account name like the desktop

### PR DESCRIPTION
**Pull request by dannywerner**
- save account name and email until registration in prefs
- save account name until login in prefs
- show account name in sidemenu
- add colorfilter for dropdown drawable in sidemenu
- add sidemenu header background drawable based on mockups
- extends test to check the correct storing of boxname and email
- add box login test

Resolves #371

### CODE REVIEW

**Comment by dannywerner** [comment]
example-account-name

File: qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/CreateBoxAccountUITest.java [hunk 1]
```diff
@@ -112,6 +113,7 @@ public void createBoxAccountTest() throws Throwable {
 
         onView(withText(R.string.create_box_account)).perform(click());
         String accountName = UUID.randomUUID().toString().substring(0, 15).replace("-", "x");
+        String accountEMail=accountName + "@example.com";
```
Comment (audax) [inline qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/CreateBoxAccountUITest.java]: style: spaces between assignments

**Comment by audax** [comment]
I guess we need a logout #405 for this to work with old installations?

**Comment by audax** [comment]
Please add a test for the empty box account name that verifies that a re-login is needed.
